### ISSUE TITLE
[Bug] Spell UI Glitch 1.20.1

### ISSUE THREAD

**Issue by Bodya33381** [iron431/Irons-Spells-n-Spellbooks#231]
### Observed behaviour

For some reason unknow reason, Spellbook UI is now bugged, and doesn't display spells correctly:
![Issue1](https://github.com/iron431/Irons-Spells-n-Spellbooks/assets/144392960/34c7f5e7-9607-4a8f-ac01-6ed54625ff57)
One of the problems is that only the selected spell is shown, not all of them and when the spell is on cooldown, the white animation cooldown is "solid" instead of being transparent (Check Expected behaviour)
![Issue1 5](https://github.com/iron431/Irons-Spells-n-Spellbooks/assets/144392960/73074967-c136-4fbd-b684-08af4a76e0a0)
There is other one which I couldn't reproduce: for some reason, a second cooldown animation happens around this area:
![issue2](https://github.com/iron431/Irons-Spells-n-Spellbooks/assets/144392960/e22986e4-37ca-41da-a0e9-814dfc42c8ba)

### Expected behaviour

Expected behavior (shots taken from CurseForge page)
![Screenshot_1](https://github.com/iron431/Irons-Spells-n-Spellbooks/assets/144392960/9505f6da-1ce2-486f-b49d-cbf31daeec98)
![Screenshot_2](https://github.com/iron431/Irons-Spells-n-Spellbooks/assets/144392960/0f3b495b-9d14-4652-9954-906d31151758)

### Steps to reproduce

It just somehow appeared after some time of usage, perhaps some oprimization mod is conflicting

### Server Type

Single Player

### Iron's Spells N Spellbooks version

1.20.1 - 2.1.1, 1.20.1 - 2.1.0, 1.20.1 - 2.0.3

### Forge version

Forge 47.2.16

### Other mods

I have created a new profile which uses only optimization mods and Iron's Spells & Spellbooks.
Here's a list of mods(using Modrinth client):

- caelus-forge-3.1.0+1.20
- curios-forge-5.4.5+1.20.1
- entityculling-forge-1.6.2-mc1.20.1
- fast-ip-ping-mc1.20.1-forge-v1.0.0
- Fastload-Reforged-mc1.20.1-3.4.0
- ferritecore-6.0.1-forge
- geckolib-forge-1.20.1-4.2.4
- ImmediatelyFast-1.2.7+1.20.2 - the conflicting mod
- irons_spellbooks-1.20.1-2.1.1
- kotlinforforge-4.7.0-all
- memoryleakfix-forge-1.17+-1.1.2
- midnightlib-forge-1.4.1
- player-animation-lib-forge-1.0.2-rc1+1.20
- rubidium-extra-0.5.3.1+mc1.20.1-build.110
- rubidium-mc1.20.1-0.7.1
- starlight-1.1.2+forge.1cda73c
- TextruesRubidiumOptions-1.0.5+mc1.20.1
- architectury-9.1.12-forge

**Comment by Luviam** [comment]
Yeah ImmediatelyFast does that at the moment with this mod. Would like to see it get fixed if possible.

**Comment by iron431** [comment]
closing for lack of version support
